src/plugins/expressions/common/execution/execution.ts

```diff
@@ -40,6 +40,23 @@ import { getByAlias } from '../util/get_by_alias';
 import { ExecutionContract } from './execution_contract';
 import { ExpressionExecutionParams } from '../service';
 
+/**
+ * AbortController is not available in Node until v15, so we
+ * need to temporarily mock it for plugins using expressions
+ * on the server.
+ *
+ * TODO: Remove this once Kibana is upgraded to Node 15.
+ */
+const getNewAbortController = (): AbortController => {
+  try {
+    return new AbortController();
+  } catch (error) {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const polyfill = require('abortcontroller-polyfill/dist/cjs-ponyfill');
+    return new polyfill.AbortController();
+  }
```

Comment:
This was a bug uncovered by functional tests which we hadn't found yet: Since `AbortController` isn't supported till Node 15 (and we are currently on v14), we need to manually apply a polyfill on the server.

Since this code is in `common`, I'm hacking around the polyfill with a try/catch.